.github/workflows/ci.yml

```diff
@@ -14,6 +14,14 @@ jobs:
       - uses: actions/checkout@v4
       - name: Checkout submodules
         run: git submodule update --init --recursive
+      - name: Cache CMake build directory
+        uses: actions/cache@v4
+        with:
+          path: |
+            build
+          key: ${{ runner.os }}-cmake-build-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-cmake-build-
```

Comment:
Is it ok that Ubuntu and Linux don't specify an arch here? (I am assuming yes because this only affects the CI runner)?

Reply:
It's just a name for the cache, had to specify on the two Mac builds to differentiate them

Reply:
> It's just a name for the cache, had to specify on the two Mac builds to differentiate them

Gotcha. thanks for clarifying!